charts/hoprd-operator/templates/crd-identity-pool.yaml

```diff
@@ -62,6 +62,18 @@ spec:
                 minReadyIdentities:
                   type: number
                   description: Number of ready instances to keep
+                funding:
+                  type: object
+                  properties:
+                    schedule:
+                      type: string
+                      description: 'Cron schedule format'
+                    nativeAmount:
+                      type: string
+                      description: 'xDai limit amount'
```

Comment:
The question is in deed really good. I was not able to use the float number, because it does not implement the Hash trait, and I couldn't manage to make it work, so I opted to use the simple way of use a string.
I will ask you for help the next version, so we can change it to float.